docker-compose.yml

```diff
@@ -118,7 +118,7 @@ services:
     tmpfs: /var/lib/mysql
 
   mysql-8-0:
-    image: mysql:8.0.22
+    image: mysql:8.0.26
```

Comment:
Possibly does make sense to keep this updated with the most recent MySQL version?